linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/configure_opensc_nss_db/bash/shared.sh

```diff
@@ -4,8 +4,11 @@
 # complexity = low
 # disruption = low
 
-PKCSSW=$(/usr/bin/pkcs11-switch)
+PKCSSW="/usr/bin/pkcs11-switch"
```

Comment:
This script is in the `opensc` package so if the package is not installed the remediation will do result in `error`.

@ggbecker do you think that installing the `opensc` package in this remediation is a good idea? Installation of `opensc` feels like a different rule, however without the package we can't perform remediation of this one.

Reply:
@mildas Probably yes, Imagine that somebody will update the rule in the future and this test will be executed by the CTF.